src/components/Transcriber.tsx

```diff
@@ -1097,10 +1097,9 @@ export function Transcriber(props: IProps) {
   };
 
   const handleAutoTranscribe = (trans: string) => {
-    const cleanTrans = trans.replace(/[0-9]+:[0-9]+.[0-9]+: /g, '');
     var curTrans: string = transcriptionRef.current?.firstChild?.value ?? '';
-    if (curTrans.includes(cleanTrans)) return;
-    const transcription = curTrans + cleanTrans;
+    if (curTrans.includes(trans)) return;
+    const transcription = curTrans + trans;
```

Comment:
original code that doesn't duplicate /v 1 is still missing

Reply:
ok, I think I put logic in to match the verse number and replace the current verse with the transcribed verse.